elastalert/schema.yaml

```diff
@@ -303,6 +303,16 @@ properties:
   kibana_discover_to_timedelta: *timedelta
   kibana_discover_security_tenant: {type:string}
 
+  ### Opensearch discover App Link
+  generate_opensearch_discover_url: {type: boolean}
+  opensearch_url: {type: string, format: uri}
+  opensearch_discover_app_url: {type: string}
+  opensearch_discover_version: {type: string, enum: ['2.11']}
+  opensearch_discover_index_pattern_id: {type: string, minLength: 1}
+  opensearch_discover_columns: {type: array, items: {type: string, minLength: 1}, minItems: 1}
+  opensearch_discover_from_timedelta: *timedelta
+  opensearch_discover_to_timedelta: *timedelta
+  
```

Comment:
add `opensearch_discover_security_tenant: {type:string}`

Reply:
add `opensearch_url: {type: string, format: uri}`

Reply:
Shouldn't settings related to Shorten URL be added?

```yaml
opensearch_username: {type: string}
opensearch_password: {type: string}
opensearch_verify_certs: {type: boolean}
```

look
[Short Kibana Discover URLs](https://github.com/jertel/elastalert2/pull/512)
[feat: add verify toggle for Kibana Shorten URL API](https://github.com/jertel/elastalert2/pull/1013)

Reply:
Hello I can't add what I have not implemented in the code the varaibles I have implemented are the one I tested

- ``opensearch_discover_security_tenant: {type:string}`` : I have not and I'm not sure the link will work with it without test.
I know I call a seucrity_tenant variable in create_opensearch_external_url_formatter but it was to be able to reuse existing function and in case I have the time to adapt it in the future, same thing goes for the shorten_url
- Since I do not call the shorten API this variables (opensearch_username, opensearch_password, opensearch_verify_certs) do not exist yet.